pages/api/chat.ts

```diff
@@ -68,8 +68,9 @@ const handler = async (req: NextRequest): Promise<Response> => {
       const token: CustomJWT = await getToken({req});
       let resp;
       try {
+        const apimUrl = `${OPENAI_API_HOST}/${APIM_CHAT_ENDPONT}/deployments/${AZURE_DEPLOYMENT_ID}/chat/completions?api-version=${OPENAI_API_VERSION}`
```

Comment:
I just refactored this for debugging, but it also makes the code (marginally) more readable